### PR TITLE
Notebook updates

### PR DESCRIPTION
**Pull request by nkhadka21**
This PR updates some of the existing notebooks. Additionally, it has applied a correction factor to the velocity dispersion function.

### CODE REVIEW

File: slsim/Deflectors/velocity_dispersion.py [hunk 1]
```diff
@@ -353,7 +353,10 @@ def schechter_vel_disp(
         resolution=1000,
         scale=1,
     )
-    return z, vel_disp
+    # above vel_disp is measured with in the effective radius/8. So, correction factor
+    # of 8**(-0.066) must be applied to convert to vel_disp at effective radius.
```
Comment (sibirrer) [inline slsim/Deflectors/velocity_dispersion.py:357]: can you provide a reference or more detailed calculation for this?

**Review by sibirrer — CHANGES_REQUESTED** [review]
Looks good @nkhadka21 I have one minor request

File: slsim/Deflectors/velocity_dispersion.py (hunk 1 above)
Comment (nkhadka21) [reply slsim/Deflectors/velocity_dispersion.py:357]: Added a reference for this correction factor.

**Review by sibirrer — APPROVED** [review]
Thank you @nkhadka21. Please merge after passing of the CIs

**Pull request merged**